opencti-platform/opencti-front/src/private/components/data/playbooks/PlaybookAddComponents.jsx

```diff
@@ -192,6 +193,7 @@ const PlaybookAddComponentsContent = ({
         { label: t_i18n('Author'), value: 'createdBy', isMultiple: false },
         { label: t_i18n('Confidence'), value: 'confidence', isMultiple: false },
         { label: t_i18n('Score'), value: 'x_opencti_score', isMultiple: false },
+        { label: t_i18n('Assignees'), value: 'objectAssignee', isMultiple: false },
```

Comment:
I would say isMultiple should be true, right ?

Reply:
You are right on this one. Changing it now !